### PR TITLE
Add UUID instances

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: src/Data/Swagger/Internal/Schema.hs [hunk 1]
```diff
@@ -433,6 +434,8 @@ instance (ToSchema a, ToSchema b) => ToSchema (Either a b)
 instance ToSchema () where
   declareNamedSchema _ = pure (NamedSchema Nothing nullarySchema)
 
+instance ToSchema UUID where declareNamedSchema = plain . paramSchemaToSchema
```
Comment (fizruk) [inline src/Data/Swagger/Internal/Schema.hs]: An example makes docs way nicer:

```
    & example ?~ toJSON UUID.nil
```

also not sure it should be unnamed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+2.1.3
+-----
+
+* Add UUID instances
```
Comment (fizruk) [inline CHANGELOG.md]: Please, mention PR.

Comment (phadej) [inline CHANGELOG.md]: done

File: src/Data/Swagger/Internal/Schema.hs (hunk 1 above)
Comment (phadej) [inline src/Data/Swagger/Internal/Schema.hs]: done.

**Comment by phadej** [comment]
travis is green

**Pull request merged**